test/trace_write_var.c

```diff
@@ -7,12 +7,10 @@
 #include <stddef.h>
 
 struct expected_entry expected[] = {
-#if !defined(__clang__)
-    EXPECT_ENTRY(COLDTRACE_ALLOC),
-    EXPECT_ENTRY(COLDTRACE_FREE),
-#endif
+    EXPECTED_SOME(COLDTRACE_ALLOC, 0, 1),
```

Comment:
An alternative to the `EXPECTED_SOME` is `EXPECT_SUFFIX(COLDTRACE_WRITE)`, which would mean, expect anything until a write happens.